applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -508,6 +531,15 @@ def __LaunchRunHROM(self, mu_run, use_full_model_part):
             simulation.Run()
             self.QoI_Run_HROM.append(simulation.GetFinalData())
 
+    def __LaunchTrainNN(self):
+        rom_nn_trainer = Rom_NN_trainer(self.general_rom_manager_parameters)
+        model_name = rom_nn_trainer.train_network()
+        rom_nn_trainer.evaluate_network(model_name)
```

Comment:
```suggestion
        model_name = rom_nn_trainer.TrainNetwork()
        rom_nn_trainer.EvaluateNetwork(model_name)
```
Please follow the Kratos [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide).

Reply:
Thank you Ruben! I'll make sure to follow it from now on